### PR TITLE
[gui] remove two not-implemented infolabels for PVR

### PR DESCRIPTION
**Pull request by phil65**
remove pvr.hastxt and pvr.hasepg since theyre not implemented.
@ksooo 	I found out that we already have videoplayer.hasteletext and videoplayer.hasepg :)

Also updated wiki already by removing wrong infolabels and adding correct ones.

I changed all define values to avoid gaps because we are looping over that range at some places.

### CODE REVIEW

**Comment by phil65** [comment]
jenkins build this please.

**Review by ksooo — APPROVED** [review]

**Comment by ksooo** [comment]
jenkins build and merge

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -4276,16 +4276,6 @@ const infomap playlist[] =       {{ "length",           PLAYLIST_LENGTH },
 ///                  _string_,
 ///     Backend number
 ///   }
-///   \table_row3{   <b>`Pvr.HasEpg`</b>,
-///                  \anchor Pvr_HasEpg
-///                  _boolean_,
-///     Returns true when an epg is available.
-///   }
-///   \table_row3{   <b>`Pvr.HasTxt`</b>,
```
Comment (AlwinEsch) [inline xbmc/GUIInfoManager.cpp]: Is this info no more used and working (Seen that Estuary not use it)?

Was thought from me to enable Teletext GUI button on xml only if Teletext is present.

Comment (phil65) [inline xbmc/GUIInfoManager.cpp]: VideoPlayer.HasTeletext seems to be the correct one ;)
Pvr.HasTxt wasnt even implemented.

Comment (AlwinEsch) [inline xbmc/GUIInfoManager.cpp]: Ups, sorry.

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
jebkins Android errors are not related to this PR

**Pull request merged**